Makefile

```diff
@@ -21,37 +21,42 @@ build:
 	go build -o build/_output/bin/picchu ./cmd/manager
 
 deps:
-	dep ensure -v
+	go mod tidy
+	go mod vendor
 
-generate: deepcopy defaulter openapi clientset
+generate: deepcopy defaulter openapi clientset crds
 
```

Comment:
just delete code, rather than commenting it out. we have git history if we need to know what used to be there.